fiftyone/plugins/secrets.py

```diff
@@ -155,5 +168,7 @@ def values(self):
         raise RuntimeError("Iteration through values is not allowed")
 
     def items(self):
-        # Override items() to prevent iterating through plaintext values
-        raise RuntimeError("Iteration through items is not allowed.")
+        # Override items() to use __getitem__ to automatically resolve
```

Comment:
`values()` should be allowed too. It's a documented as a dict, so users should be allowed to do dict-like things

Reply:
I'm against this, but fine